content/docs/development/oidc-certification.md

```diff
@@ -0,0 +1,154 @@
+| oidcc-ensure-request-without-nonce-succeeds-for-code-flow                    | FINISHED    | PASSED  |
+| oidcc-scope-profile                                                          | FINISHED    | WARNING |
+| oidcc-scope-email                                                            | FINISHED    | WARNING |
+| oidcc-scope-address                                                          | FINISHED    | SKIPPED |
+| oidcc-scope-phone                                                            | FINISHED    | SKIPPED |
+| oidcc-scope-all                                                              | FINISHED    | SKIPPED |
+| oidcc-ensure-other-scope-order-succeeds                                      | FINISHED    | WARNING |
+| oidcc-display-page                                                           | FINISHED    | PASSED  |
+| oidcc-display-popup                                                          | FINISHED    | PASSED  |
+| oidcc-prompt-login                                                           | INTERRUPTED | UNKNOWN |
+| oidcc-prompt-none-not-logged-in                                              | FINISHED    | FAILED  |
+| oidcc-prompt-none-logged-in                                                  | FINISHED    | PASSED  |
+| oidcc-max-age-1                                                              | INTERRUPTED | FAILED  |
+| oidcc-max-age-10000                                                          | FINISHED    | FAILED  |
+| oidcc-ensure-request-with-unknown-parameter-succeeds                         | FINISHED    | PASSED  |
+| oidcc-id-token-hint                                                          | FINISHED    | PASSED  |
+| oidcc-login-hint                                                             | FINISHED    | PASSED  |
+| oidcc-ui-locales                                                             | FINISHED    | PASSED  |
+| oidcc-claims-locales                                                         | FINISHED    | PASSED  |
+| oidcc-ensure-request-with-acr-values-succeeds                                | FINISHED    | WARNING |
+| oidcc-codereuse                                                              | FINISHED    | PASSED  |
+| oidcc-codereuse-30seconds                                                    | FINISHED    | WARNING |
+| oidcc-ensure-registered-redirect-uri                                         | INTERRUPTED | REVIEW  |
+| oidcc-server-client-secret-post                                              | FINISHED    | PASSED  |
+| oidcc-unsigned-request-object-supported-correctly-or-rejected-as-unsupported | INTERRUPTED | UNKNOWN |
+| oidcc-claims-essential                                                       | FINISHED    | WARNING |
+| oidcc-ensure-request-object-with-redirect-uri                                | INTERRUPTED | UNKNOWN |
+| oidcc-refresh-token                                                          | INTERRUPTED | FAILED  |
```

Comment:
This one should be fixed by https://github.com/dexidp/dex/commit/578cb05f7b160bffa432a9c6b1e571744a01b660